courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>Instead of this cell and the three cells below, I think a more concise way of displaying the same information is to do</p><p>!head *.csv</p><p> </p><p>This is similar to how it's done in previous labs as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Nice!  Removed the "print output" cell and modified the output to csv files.  Added the "!head *.csv".</p><p> </p>